services/execution/service.go

```diff
@@ -129,7 +130,7 @@ func (s *Service) Create(ctx context.Context, r *api.CreateRequest) (*api.Create
 	}
 	c, err := runtime.Create(ctx, r.ContainerID, opts)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "runtime create failed")
```

Comment:
This is the service method implementation. Seems like the right place to do it...

We'll cover this in another round of review.